protocols/69db93/README.md

```diff
@@ -0,0 +1,89 @@
+# RNA Extraction with Magnetic Beads (no tip waste)
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+## Categories
+* NUCLEIC ACID EXTRACTION & PURIFICATION
```

Comment:
I think capitalization may affect how this shows up on the protocol library -- I may be wrong, but I think using different capitalization will create a new top-level category here